docs/concepts/protocol/07-airstreams.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: "airstreams"
+sidebar_position: 7
+title: "Airstreams"
+---
+
+:::note
+
+Please refer to the [airdrop section](/concepts/use-cases#airdrops) of our use-cases page for an explanation regarding
+why streaming an airdrop makes sense.
+
+:::
+
+## What are Airstreams?
+
+The gist of Airstreams is that instead of airdropping the entirety of the tokens all at once, airdrop recipients receive
+a fraction of the tokens every second through a token stream.
+
+The beauty of it is that airdrop recipients are forced to think long-term and keep the project’s future as their first
+and foremost priority. They are forced to, because instead of receiving all the tokens at once, they receive them over
+time.The gorgeous Airstreams interface
```

Comment:
Doesn't it answer "why Airstreams" over "What Airstreams"? 

May be something like :-
Airstreams is a new platform that let you create an airdrop program for your users where tokens are streamed to them over a period of time instead of all at once, similar to how vesting works.

And then we go about why it is a useful concept.

Reply:
yes, you are right, we should change the chapter to "## Why Airstreams" instead of "## What are Airstreams?"